go.mod

```diff
@@ -3,19 +3,23 @@ module github.com/celestiaorg/celestia-node
 go 1.23.6
 
 require (
-	cosmossdk.io/math v1.5.0
+	cosmossdk.io/log v1.5.1
+	cosmossdk.io/math v1.5.3
+	cosmossdk.io/store v1.1.2
+	cosmossdk.io/x/feegrant v0.1.1
 	github.com/BurntSushi/toml v1.4.1-0.20240526193622-a339e1f7089c
 	github.com/alecthomas/jsonschema v0.0.0-20220216202328-9eeeec9d044b
 	github.com/benbjohnson/clock v1.3.5
-	github.com/celestiaorg/celestia-app/v3 v3.8.0-arabica
+	github.com/celestiaorg/celestia-app/v4 v4.0.0-20250415151309-5138040df8a7
```

Comment:
should not be merged until there is a tag, unless we are okay with a follow up PR to do this.